specification/maps/data-plane/readme.md

```diff
@@ -47,88 +46,3 @@ directive:
     reason: false positive from oav is breaking our example validation. See azure/oav#1021.
 
 ```
-
-### Tag: package-stable-2023-06-01
```

Comment:
This has to be added along side with the changes that add those reference in service readme.md file